_ext/events.rb

```diff
@@ -0,0 +1,51 @@
+require 'aweplug/helpers/searchisko'
+require 'parallel'
+
+module JBoss
+  module Developer
+    class Events
+
+      def initialize
+
+      end
+
+      def execute site
+        searchisko = Aweplug::Helpers::Searchisko.default site, 0
+
+        Parallel.each(site.events, in_threads: (site.build_threads || 0)) do |(id, data)|
+
+        searchisko_hash = data.collect { |(key, value)|
```

Comment:
This whole { ... } block, please switch it to use the do ... end syntax. The { ... } syntax is for one liners.